bullet_train-themes/app/views/themes/base/attributes/_date.html.erb

```diff
@@ -1,11 +1,12 @@
 <% object ||= current_attributes_object %>
 <% strategy ||= current_attributes_strategy || :none %>
 <% url ||= nil %>
+<% local_assigns[:format] ||= :default %>
```

Comment:
iirc, `local_assigns` isn't meant to be mutated, plus we can use the same format as the other attributes:

```suggestion
<% format ||= :default %>
```